streampipes-client-go/streampipes/internal/streampipes_http/delete_request.go

```diff
@@ -0,0 +1,79 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package streampipes_http
+
+import (
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/config"
+	"github.com/apache/streampipes/streampipes-client-go/streampipes/internal/statu_code"
+	"io"
+	"log"
+	"net/http"
+)
+
+type DeleteRequest struct {
+	HttpRequest *httpRequest
+}
+
+var _ HttpRequest = (*DeleteRequest)(nil)
+
+func NewDeleteRequest(config config.StreamPipesClientConnectConfig) *DeleteRequest {
+	return &DeleteRequest{
+		HttpRequest: NewHttpRequest(config),
+	}
+}
+
+func (d *DeleteRequest) ExecuteRequest(model interface{}) interface{} {
+	d.makeRequest()
+
+	if d.HttpRequest.Response.StatusCode == 200 {
+		return nil
+	} else {
+		switch d.HttpRequest.Response.StatusCode {
+		case statu_code.BadRequest.Code():
+			log.Fatal(statu_code.BadRequest.Code(), statu_code.BadRequest.Message())
+		case statu_code.Unauthorized.Code():
+			log.Fatal(statu_code.Unauthorized.Code(), statu_code.Unauthorized.Message())
+		case statu_code.AccessDenied.Code():
+			log.Fatal(statu_code.AccessDenied.Code(), statu_code.AccessDenied.Message())
+		case statu_code.MethodNotAllowed.Code():
+			log.Fatal(statu_code.MethodNotAllowed.Code(), statu_code.MethodNotAllowed.Message())
+		default:
+			defer d.HttpRequest.Response.Body.Close()
+			body, _ := io.ReadAll(d.HttpRequest.Response.Body)
+			log.Fatal(d.HttpRequest.Response.Status, string(body))
```

Comment:
I think this logic could be abstracted.

Reply:
If the error_code module is not required, error messages can be customized for each request. Maybe there's no need to abstract it.